src/data/BBFields.py

```diff
@@ -44,11 +45,14 @@ def set_fields_from_dict(self, d):
                 setattr(self, param, [])
             return
 
+        if "accident_locations" not in d:
```

Comment:
(as mentioned in discord) I think it makes more sense to just put accident_locations outside of bbox_fields and directly inside metadataitem if possible just cause it's not structured like the rest of the data where theres one entry per bounding box